src/platform/packages/shared/kbn-scout/src/playwright/fixtures/test/performance/README.md

```diff
@@ -0,0 +1,126 @@
+## Performance Tracker Fixture
+
+### Overview
+
+`perfTrackerFixture` is a Playwright fixture designed to analyze JavaScript bundle performance and page-level performance metrics in Kibana by leveraging Chrome DevTools Protocol (CDP). It intercepts network requests, filters static bundles, and computes bundle size statistics per page load. Additionally, it collects CDP Performance Domain Metrics, allowing in-depth analysis of rendering and script execution times.
```

Comment:
I love that the `perfTracker` fixture has a dedicated README document (and it's a wonderful read, too).

I wonder if we could do something similar for the Lighthouse fixture, if you think it's a good idea and/or worth explaining.

Reply:
Good point, I will push a small readme as well